src/Samples/Toolkit.SampleApp.Maui/MainPage.xaml

```diff
@@ -36,8 +36,8 @@
                         <Label MaxLines="5" Margin="0,10,0,0" >
                             <Label.FormattedText>
                                 <FormattedString>
-                                    <Span Text="You can get or create an API key from " />
-                                    <Span Text="your developer dashboard" TextDecorations="Underline" TextColor="{AppThemeBinding Light={StaticResource Blue100Accent}, Dark={StaticResource Blue300Accent}}" >
+                                    <Span Text="You can get or create an API key by following" />
```

Comment:
You can create an API Key using the Create an api key Tutorial... but you cannot get/copy one after the API Key has been generated. You only get one opportunity to copy it so I would be tempted to just say "You can create an API Key access token by...."